### ISSUE TITLE
"Pick Up As Item" doesn't add the armor stand to inventory

### ISSUE THREAD

**Issue by Frumple** [St3venAU/ArmorStandTools#67]
After our server upgraded to Paper 1.18.1 (build 145) and ArmorStandTools 4.4.2, the "Pick Up As Item" tool no longer adds the armor stand to your inventory. This happens in either creative or survival mode (armor stand is removed from the world in survival as normal). When using the tool, the console reports the following error:

```
[16:56:04 INFO]: CONSOLE issued server command: /give Frumple minecraft:armor_stand{Enchantments:[{id:unbreaking,lvl:1}],HideFlags:1,display:{Name:"\"Configured Armor Stand\"",Lore:["\"Inventory: 4 Items (4 Stacks)\"","\"Attributes: Gravity Off, Arms\""]},EntityTag:{NoBasePlate:1,NoGravity:1,ShowArms:1,Rotation:[357.45f],Pose:{Body:[0.0f,0.0f,0.0f],Head:[0.0f,0.0f,0.0f],LeftLeg:[-1.0f,0.0f,-1.0f],RightLeg:[1.0f,0.0f,1.0f],LeftArm:[-10.0f,0.0f,-10.0f],RightArm:[-15.0f,0.0f,10.0f]}}}
[16:56:04 INFO]: Error: Unknown item name: minecraft:armor_stand{enchantments:[{id:unbreaking,lvl:1}],hideflags:1,display:{name:"\"configured.
```

I suspect there's something going wrong with the double quotes, since the /give command cuts off as soon as it encounters a space character. But other than that, I'm not sure what the cause is.

**Comment by St3venAU** [comment]
I just tried with Paper 1.18.1 build 152 and did not get an error. Could you try the latest paper build?

**Comment by Frumple** [comment]
I updated to Paper build 152, but got the same result.

However, I think I know the cause of the issue now. We have **EssentialsX** installed on our server, and it overrides the vanilla `/give` command with its own, and that command doesn't seem to like those double quotes.

I did a little test and can confirm that these commands using `/give` return the "Unknown item name" error:
```
/give Frumple minecraft:diamond_sword{display:{Lore:["\"A legendary weapon\""]}} 1
/give Frumple minecraft:armor_stand{Enchantments:[{id:unbreaking,lvl:1}],HideFlags:1,display:{Name:"\"Configured Armor Stand\"",Lore:["\"Inventory: 4 Items (4 Stacks)\"","\"Attributes: Gravity Off, Arms\""]},EntityTag:{NoBasePlate:1,NoGravity:1,ShowArms:1,Rotation:[357.45f],Pose:{Body:[0.0f,0.0f,0.0f],Head:[0.0f,0.0f,0.0f],LeftLeg:[-1.0f,0.0f,-1.0f],RightLeg:[1.0f,0.0f,1.0f],LeftArm:[-10.0f,0.0f,-10.0f],RightArm:[-15.0f,0.0f,10.0f]}}}
```
But these commands do work if they use `/minecraft:give` instead:
```
/minecraft:give Frumple minecraft:diamond_sword{display:{Lore:["\"A legendary weapon\""]}} 1
/minecraft:give Frumple minecraft:armor_stand{Enchantments:[{id:unbreaking,lvl:1}],HideFlags:1,display:{Name:"\"Configured Armor Stand\"",Lore:["\"Inventory: 4 Items (4 Stacks)\"","\"Attributes: Gravity Off, Arms\""]},EntityTag:{NoBasePlate:1,NoGravity:1,ShowArms:1,Rotation:[357.45f],Pose:{Body:[0.0f,0.0f,0.0f],Head:[0.0f,0.0f,0.0f],LeftLeg:[-1.0f,0.0f,-1.0f],RightLeg:[1.0f,0.0f,1.0f],LeftArm:[-10.0f,0.0f,-10.0f],RightArm:[-15.0f,0.0f,10.0f]}}}
```

So I think the fix is simply using `/minecraft:give` instead of `/give`, by changing this one line: https://github.com/St3venAU/ArmorStandTools/blob/master/src/main/java/com/gmail/St3venAU/plugins/ArmorStandTools/Utils.java#L274. I've created pull request #69 for this, if you could take a look that would be great. Thanks!

**Comment by St3venAU** [comment]
Thanks. This change is now in the just released v4.4.3.
